src/id3/v2/frame/mod.rs

```diff
@@ -173,6 +174,8 @@ pub enum FrameValue {
 	Picture(AttachedPictureFrame),
 	/// Represents a "POPM" frame
 	Popularimeter(Popularimeter),
+	/// Represents an "IPLS" or "TPIL" frame
+	KeyValueFrame(KeyValueFrame),
```

Comment:
```suggestion
	KeyValue(KeyValueFrame),
```
This should follow the other variants' definitions: `X(XFrame)`